mobile/cmd/main.go

```diff
@@ -2,11 +2,11 @@ package main
 
 import (
 	"fmt"
-	"os"
-	"sync"
-
 	"github.com/OpenBazaar/openbazaar-go/mobile"
 	"github.com/jessevdk/go-flags"
+	"os"
+	"sync"
+	"time"
```

Comment:
Let's keep the packages organized consistently with newlines between each group:
```
import (
  // stdlib_packages

  // external_packages
 
  // gx_packages
)
```

Reply:
Doesn't seem to be anyway around this. Gofmt always moves it into this position.

Reply:
It will if you include the line breaks as shown above.